apps/total-typescript/package.json

```diff
@@ -20,7 +20,7 @@
     "@amplitude/analytics-browser": "^1.3.0",
     "@amplitude/identify": "^1.10.2",
     "@amplitude/node": "^1.10.2",
-    "@casl/ability": "^6.0.0",
+    "@casl/ability": "^6.3.2",
```

Comment:
bumping this to latest

v6 has an alternative syntax for defining abilities so I updated that as well https://casl.js.org/v6/en/advanced/typescript